<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UAVControl</class>
 <widget class="QWidget" name="UAVControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addUavButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>18</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new UAV</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="uavIdBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>18</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QListView" name="uavListView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>571</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>21</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="radiusSpinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>95</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>77</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Radius</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>75</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Frame</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="timeFrameSpinBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>95</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="colAvEnabledCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>95</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="changeParamsButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>85</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>61</y>
     <width>571</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>73</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Collision Avoidance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>78</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sync zone</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="syncZoneSpinBox">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>95</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>6</number>
   </property>
  </widget>
  <widget class="QPushButton" name="loadScenarioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Scenario</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveScenarioButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Scenario</string>
   </property>
  </widget>
  <widget class="QPushButton" name="simActionButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>16</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start sim</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
